common/test/acceptance/tests/helpers.py

```diff
@@ -139,7 +139,7 @@ def enable_jquery_animations(page):
 
 def disable_css_animations(page):
     """
-    Disable CSS3 animations, transitions, transforms.
+    Disable CSS3 animations and transitions.
```

Comment:
This seems like a change that would break other tests. Can you instead just use a parameter or decorator so that `disable_animations(page)` is not called for the specific tests you need?

Or, better yet: Instead of setting an absolute `top` position and using `transform: translateY(-100%)`, just leave `top` undefined and set an absolute `bottom` position. Yeah, do that.